supplier_api_tester/supplier_api_tester/utils/availability.py

```diff
@@ -43,6 +43,59 @@ def test_next_30_days(api_url, api_key, product_id, endpoint, adapter_func, vers
 
     return TestResult()
 
+def test_30_days_single_timeslots(api_url, api_key, product_id, endpoint, adapter_func, version=1):
+    '''Checking timeslots'''
+    start = datetime.utcnow().date()
+    end = start + timedelta(days=30)
+
+    raw_response, response = client(f'{api_url}/v{version}/products/{product_id}/{endpoint}', api_key, {
+        'start': start.isoformat(),
+        'end': end.isoformat(),
+    })
+
```

Comment:
First parse the results and turn them into a list of `Timeslot` objects:
```py
timeslots = adapter_func(raw_response, response)
```
then use them as objects.